proof/refine/ARM/Reply_R.thy

```diff
@@ -96,4 +95,469 @@ lemma replyRemoveTCB_st_tcb_at'_sym_ref:
   apply (clarsimp simp: obj_at'_def pred_tcb_at'_def)
   done
 
+crunches setReplyTCB
+  for ksReadyQueues[wp]: "\<lambda>s. P (ksReadyQueues s)"
+  and ksSchedulerAction[wp]: "\<lambda>s. P (ksSchedulerAction s)"
+  and valid_queues[wp]: "valid_queues"
+
+crunches getReplyTCB
```

Comment:
Add "FIXME RT: move"

Reply:
Anywhere it "should" go?

Reply:
No idea. If there is not a general place for it, then at least put it at the top of the file?